.chglog/CHANGELOG.tpl.md

```diff
@@ -50,4 +50,4 @@
 [{{ .Tag.Name }}]: {{ $.Info.RepositoryURL }}/compare/{{ .Tag.Previous.Name }}...{{ .Tag.Name }}
 {{ end -}}
 {{ end -}}
-{{ end -}}
\ No newline at end of file
+{{ end -}}
```

Comment:
What‘s the change here?

Reply:
This was the result of applying `insert_final_newline = true` from `.editorconfig`.